test/mobile/features/ios/ios_enabledbreadcrumbs.feature

```diff
@@ -0,0 +1,11 @@
+Feature: android enabled breadcrumbs
```

Comment:
```suggestion
Feature: ios enabled breadcrumbs
```

?

Why have separate files for enabled vs max breadcrumbs? Seems reasonable to have one `breadcrumbs.feature` for anything related.

Reply:
done